src/components/theme/ThemeProvider.tsx

```diff
@@ -30,6 +32,7 @@ export interface ThemeProviderProps extends React.PropsWithChildren<{}> {
     scoped?: boolean;
     rootClassName?: string;
     layout?: Omit<PrivateLayoutProviderProps, 'children'>;
+    langOptions?: Partial<LangOptions>;
```

Comment:
I would keep `lang` and `fallbackLang` on the top level to not be repetitive (`langOptions.lang*`)